### PR TITLE
Added static screens

### PR DESCRIPTION
**Pull request by shivanigangadharan**
Added static screens (with dummy data) for the following pages :- 
- Homepage
- Watch later
- History
- Playlists list page
- Individual playlist page

### CODE REVIEW

File: src/pages/playlists/playlists.js [hunk 1]
```diff
@@ -1,9 +1,23 @@
...
-        <div>
-            <h1> Playlists </h1>
+        <div className="videolisting-container">
+            <div className="history-title">
+                <h1 className="bold"> My Playlists </h1>
+                <div>
+                    <button className="btn clear-history"> Create new playlist </button>
+                </div>
+            </div>
+            <div className="video-grid">
```
Comment (ajit-sharmaGH) [inline src/pages/playlists/playlists.js:15]: Good naming, easily readable

**Review by ajit-sharmaGH — APPROVED** [review]
Code looks clean
Easy to get the code structure 
Just a suggestion try to add pics during you raise PR 
everything  looks good 
You can merge it

File: src/pages/videolisting/videolisting.css [hunk 2]
```diff
@@ -1,10 +1,8 @@
 .videolisting-container{
     padding: 100px;
```
Comment (deepak29-git) [inline src/pages/videolisting/videolisting.css]: use rem

File: src/pages/homepage/homepage.css [hunk 3]
```diff
@@ -0,0 +1,33 @@
+.homepage-container{
+    padding: 100px;
```
Comment (deepak29-git) [inline src/pages/homepage/homepage.css]: use rem unit for achieve responsiveness

**Review by deepak29-git — APPROVED** [review]

File: src/pages/homepage/homepage.css (hunk 3 above)
Comment (shivanigangadharan) [reply src/pages/homepage/homepage.css]: Done [21c1c43](https://github.com/shivanigangadharan/streambucket/pull/4/commits/21c1c43d5b49d9b08c2469a473d8cb79db95cb87)

File: src/pages/videolisting/videolisting.css (hunk 2 above)
Comment (shivanigangadharan) [reply src/pages/videolisting/videolisting.css]: Done [21c1c43](https://github.com/shivanigangadharan/streambucket/pull/4/commits/21c1c43d5b49d9b08c2469a473d8cb79db95cb87)

**Pull request merged**